regtest.sh

```diff
@@ -343,17 +322,16 @@ default-processes() {
 # Args:
 #   spec_gen: A program to execute to generate the spec.
 #   spec_regex: A pattern selecting the subset of tests to run
-#   instances: A number of times each test case is run
 #   parallel: Whether the tests are run in parallel (T/F).  Sequential
 #     runs log to the console; parallel runs log to files.
-#   impl: one of fast_counts, python, cpp
+#   impl: one of python, or cpp
 
 _run-tests() {
   local spec_gen=$1
   local spec_regex="$2"  # grep -E format on the spec, can be empty
-  local instances=$3
-  local parallel=$4
-  local impl=$5
+  local parallel=$3
+  local impl=$4
+  local instances=1
```

Comment:
It runs the the same test case more than once.  By default the regtest does each thing 5 times (but the demo doesn't).

Ilya specifically added this at one point, so I assume he wants it.